store/cli.go

```diff
@@ -38,12 +38,13 @@ func CLIFlags(envPrefix, category string) []cli.Flag {
 			Category: category,
 			Required: false,
 		},
-		&cli.BoolFlag{
-			Name:     DisperseToV2FlagName,
-			Usage:    "Enable blob dispersal to EigenDA V2 protocol.",
-			EnvVars:  withEnvPrefix(envPrefix, "DISPERSE_TO_V2"),
+		&cli.StringFlag{
+			Name:     DispersalBackendFlagName,
+			Usage:    "Target EigenDA backend version for blob dispersal (e.g. V1 or V2).",
+			EnvVars:  withEnvPrefix(envPrefix, "DISPERSAL_BACKEND"),
 			Category: category,
 			Required: false,
+			Value:    "V1",
 		},
```

Comment:
using a generic flag to implement the actual enum would be nice: see https://github.com/urfave/cli/issues/602#issuecomment-416000118

Reply:
See [this comment](https://github.com/Layr-Labs/eigenda-proxy/pull/361#discussion_r2035783143)